src/game/shell.c

```diff
@@ -199,9 +199,21 @@ bool Shell_MakeScreenshot()
         break;
     }
 
-    char path[20];
+    char path[120];
     for (int i = 0; i < 10000; i++) {
-        sprintf(path, "screenshot%04d.%s", i, ext);
+#if defined(_WIN32)
+    #include <direct.h>
+        _mkdir("screenshots");
+#else
+    #include <sys/types.h>
+    #include <sys/stat.h>
+        mkdir("screenshots", 0664);
+#endif
```

Comment:
We don't do `#ifdefs` in `game/`. Also this logic doesn't belong here. Please make a `S_File_CreateDirectory` function and create a proxy for it in `filesystem.c` that gets called here.

Reply:
Ok. Ngl I’m not sure how a proxy works.

Reply:
Proxy in context of T1M is when we avoid inclusion of platform-specific code by game modules. Example: `File_GetGameDirectory` is a proxy for `S_File_GetGameDirectory`.

This concept is less than ideal and I want to do without it at some point, but for now it's better to continue this single concept rather than coming up with new ones to track.